parser-typechecker/src/Unison/Builtin.hs

```diff
@@ -560,6 +560,7 @@ ioBuiltins =
   , ("IO.putBytes.impl.v3", handle --> bytes --> iof unit)
   , ("IO.getLine.impl.v1", handle --> iof text)
   , ("IO.systemTime.impl.v3", unit --> iof nat)
+  , ("IO.systemTime2.impl.v1", unit --> iof int)
```

Comment:
How about we call this `IO.systemTimeMicroseconds.v1`? I think the return result can just be `io int` too, since this function can't fail. You can then simplify the calling convention.

We use the `.impl` convention only when the builtin has to return an `Either`, so that the nicely named version returns `{Exception,IO} a`. 

The `.v1` will be stripped out of the default name that shows up when you `builtins.merge`, and we'll add an alias for all new builtins in base when we do the next release.

Reply:
Done